src/Network/Network/help/Az.Network.md

```diff
@@ -299,9 +299,6 @@ Gets a Azure Firewall.
 ### [Get-AzFirewallFqdnTag](Get-AzFirewallFqdnTag.md)
 Gets the available Azure Firewall Fqdn Tags.
 
-### [Get-AzInterfaceEndpoint](Get-AzInterfaceEndpoint.md)
-The Get-AzInterfaceEndpoint cmdlet gets a Interface Endpoint.
-
```

Comment:
It looks like this file wasn't regenerated after the new cmdlets were added. Please regenerate. When you run `Update-MarkdownHelpModule`, make sure to include the `-RefreshModulePage` parameter.

Reply:
Re-generated the file. Thank you.